pkg/controller/servingcert/controller/secret_creating_controller.go

```diff
@@ -407,6 +409,7 @@ func toRequiredSecret(dnsSuffix string, ca *crypto.CA, intermediateCACert *x509.
 
 	secretCopy.Annotations[api.AlphaServingCertExpiryAnnotation] = servingCert.Certs[0].NotAfter.Format(time.RFC3339)
 	secretCopy.Annotations[api.ServingCertExpiryAnnotation] = servingCert.Certs[0].NotAfter.Format(time.RFC3339)
+	secretCopy.Annotations[api.OwningJiraComponent] = api.OwningJiraComponent
```

Comment:
Shouldn't it be `secretCopy.Annotations[apiannotations.OpenShiftComponent]` instead?